hardware/Makefile

```diff
@@ -250,7 +250,7 @@ action_hw: prepare_logs
 			echo -e "                        Error: please look into $(LOGS_DIR)/action_make.log"; exit -1; \
 		fi; \
 	fi
-	@echo -e "[CONFIG ACTION HW....] done  `date +"%T %a %b %d %Y"`";
+	@echo -e "[COMPILE ACTION HW...] done  `date +"%T %a %b %d %Y"`";
```

Comment:
See comment above

Reply:
ok got it so let's keep it as previously